agent/stats/engine.go

```diff
@@ -390,6 +390,27 @@ func (engine *DockerStatsEngine) containerHealthsToMonitor() bool {
 	return len(engine.tasksToHealthCheckContainers) != 0
 }
 
+func (engine *DockerStatsEngine) stopTrackingContainer(container *StatsContainer, taskARN string) bool {
+	terminal, err := container.terminal()
+	if err != nil {
+		// Error determining if the container is terminal. This means that the container
+		// id could not be resolved to a container that is being tracked by the
+		// docker task engine. If the docker task engine has already removed
+		// the container from its state, there's no point in stats engine tracking the
+		// container. So, clean-up anyway.
+		seelog.Warnf("Error determining if the container %s is terminal, removing from stats, err: %v", container.containerMetadata.DockerID, err)
+		engine.doRemoveContainerUnsafe(container, taskARN)
+		return true
+	} else if terminal {
```

Comment:
can you make just an `if terminal` instead? You don't need an `else if` because of the `return true` statement